test/e2e/components/vl-map-select-action.js

```diff
@@ -0,0 +1,20 @@
+const VlMapAction = require('./vl-map-action');
+const VlMapLayerCircleStyle = require('./vl-map-layer-circle-style');
+
+class VlMapSelectAction extends VlMapAction {
+
+    async isClustered() {
+        return this.hasAttribute('cluster');
+    }
+
+    async hasCircleStyle() {
+        return (await this.findElements(By.css('vl-map-layer-circle-style'))).length > 0;
```

Comment:
leest iets makkelijker als dat in een aparte variabele zit

Reply:
idem opmerking als bij layer. door hier "hasCircleStyle" en "getCircleStyle" te steken, moet ge hier binnenkort ook "hasVierkanteStijl" en "hasVeryCoolStyle" steken, dus die dependency moet je proberen om te draaien.